enterprise_access/apps/content_assignments/tasks.py

```diff
@@ -131,3 +132,81 @@ def send_cancel_email_for_pending_assignment(cancelled_assignment_uuid):
+    subsidy_policy_model = apps.get_model('subsidy_access_policy.SubsidyAccessPolicy')
+
+    try:
+        assignment = learner_content_assignment_model.objects.get(uuid=new_assignment_uuid)
+    except learner_content_assignment_model.DoesNotExist:
+        logger.warning(f'request with uuid: {new_assignment_uuid} does not exist.')
+        return
+    learner_content_assignment_action = LearnerContentAssignmentAction(
+        assignment=assignment, action_type=AssignmentActions.NOTIFIED
+    )
+
+    try:
+        policy = subsidy_policy_model.objects.get(
+            assignment_configuration=assignment.assignment_configuration
+        )
+    except subsidy_policy_model.DoesNotExist:
+        logger.warning(f'policy with assignment config: {assignment.assignment_configuration} does not exist.')
+        return
+
+    braze_trigger_properties = {}
+    braze_client_instance = BrazeApiClient()
+    lms_client = LmsApiClient()
+    enterprise_customer_uuid = assignment.assignment_configuration.enterprise_customer_uuid
+    enterprise_customer_data = lms_client.get_enterprise_customer_data(enterprise_customer_uuid)
+
+    course_metadata = get_content_metadata_for_assignments(
+        policy.catalog_uuid, [assignment]
+    )
+    lms_user_id = assignment.lms_user_id
+    admin_emails = [user['email'] for user in enterprise_customer_data['admin_users']]
+    braze_trigger_properties['contact_admin_link'] = braze_client_instance.generate_mailto_link(admin_emails)
+
+    try:
+        recipient = braze_client_instance.create_recipient(
+            user_email=assignment.learner_email,
+            lms_user_id=assignment.lms_user_id
+        )
+        braze_trigger_properties["organization"] = enterprise_customer_data['name']
+        braze_trigger_properties["course_title"] = assignment.content_title
+        braze_trigger_properties["enrollment_deadline"] = course_metadata['normalized_metadata']['enroll_by_date']
+        braze_trigger_properties["start_date"] = course_metadata['normalized_metadata']['start_date']
+        braze_trigger_properties["course_partner"] = _get_course_partners(course_metadata)
+        braze_trigger_properties["course_card_image"] = course_metadata['card_image_url']
+
+        braze_client_instance.send_campaign_message(
+            settings.BRAZE_ASSIGNMENT_NOTIFICATION_CAMPAIGN,
+            recipients=[recipient],
+            trigger_properties=braze_trigger_properties,
+        )
+        learner_content_assignment_action.completed_at = datetime.now()
+        learner_content_assignment_action.save()
+        logger.info(f'Sending braze campaign message for new assignment {assignment}')
+        return
+    except Exception as exc:
+        logger.error(f"Unable to send email for {lms_user_id} due to exception: {exc}")
+        learner_content_assignment_action.error_reason = AssignmentActionErrors.EMAIL_ERROR
+        learner_content_assignment_action.traceback = exc
```

Comment:
nit: when https://github.com/openedx/enterprise-access/pull/331 merges, there will be a function called `format_traceback` you can use here `learner_content_assignment_action.traceback = format_traceback(exc)`